app/models/concerns/tagable.rb

```diff
@@ -2,10 +2,12 @@ module Tagable
   extend ActiveSupport::Concern
 
   def tags=(json)
-    tags_params = JSON.parse(json)
-    tags_params.each do |tag_param|
-      tag = Tag.find_or_create_by(title: tag_param['value'])
-      self.tag_references.find_or_initialize_by(tag: tag)
+    if json.present?
```

Comment:
Your time has come.
Izlasi par "guard clause in Ruby" un turpmak šados gadijumos izmanto to